app/helpers/application_helper.rb

```diff
@@ -11,12 +11,11 @@ def full_title(page_title = '')
   end
 
   def flash_class(level)
-    case level
-      when :notice then "alert alert-info"
-      when :success then "alert alert-success"
-      when :error then "alert alert-error"
-      when :alert then "alert alert-error"
-    end
+    level = {
+        :notice => "alert alert-info",
+        :success => "alert alert-success",
+        :error => "alert alert-error",
+        :alert => "alert alert-error"
```

Comment:
Looking good, also FYI, when you have symbols as your keys in a hash, you can format your hash like this:
```ruby
{ key: value }
```

If you're using anything else than a symbol, you do have to use the `=>` syntax though.

Reply:
Oh cool! I'll switch it over to that then! Thank you. :D